core/services/vrf/listener_v2.go

```diff
@@ -689,6 +685,14 @@ func (lsn *listenerV2) JobID() int32 {
 	return lsn.job.ID
 }
 
+func (lsn *listenerV2) fromAddresses() []common.Address {
+	var addresses []common.Address
+	for _, a := range lsn.job.VRFSpec.FromAddresses {
+		addresses = append(addresses, a.Address())
+	}
+	return addresses
+}
```

Comment:
Should we just cache this in the `listenerV2` object upon construction? These addresses won't change throughout the lifetime of the object, since for us to change them, we would have to destroy + recreate the job (which would shut down an existing listener and bring a new one up).

Reply:
Yeah considered that, I just ended up doing it this way for simplicity, in practice this will only be a small handful of addresses so should be negligible compared to the other work involved, and wanted to avoid extra complexity by caching it